public/pickup_delivery.html.erb

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Mapotempo Optimizer - Pickup Delivery</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <div class="block">
+    <img src="images/logo.png">
+    <h1>Optimizer</h1>
+  </div>
+  <div class="block">
+    <fieldset id="optim-list-status">
+      <legend id="optim-list-legend"></legend>
+      <div id="jobs-list">
+
+      </div>
+    </fieldset>
+  </div>
+  <form>
+    <a id="doc-link" href="#documentation">documentation</a>
+    <div class="block">
+      <span id="file-customers-label"><%= I18n.t('views.shared.file_customer_label') %></span> <input type="file" id="file-customers">
+    </div>
+    <div class="block">
+      <span id="file-vehicles-label"><%= I18n.t('views.shared.csv_vehicles_label') %></span> <input type="file" id="file-vehicles">
+    </div>
+    <div class="block" id="optim-options">
+      <fieldset>
+        <legend id="optim-options-legend"><%= I18n.t('views.pickup_delivery.optim_options_legend') %></legend>
+        <div><span id="optim-duration-label"><%= I18n.t('views.pickup_delivery.optim_duration_label') %></span> <input type="text" id="optim-duration" value="00:10:00" placeholder="hh:mm:ss"></div>
+        <div><span id="optim-duration-label"><%= I18n.t('views.pickup_delivery.optim_duration_label') %></span> <input type="text" id="optim-minimum-duration" value="00:10:00" placeholder="hh:mm:ss"></div>
```

Comment:
the minimum duration label should be different from optim-duration

Reply:
Oh forgot to change it